src/plugins/python/CMakeLists.txt

```diff
@@ -0,0 +1,97 @@
+# NOTE: this plugins requires python 3 although it's compatible with python2
+# because of the following build systems issues:
+# - it's impossible to compile this plugin twice
+# - however it's also impossible to create a python2 plugin and include the
+#   cpp files of this plugin because we also need to mangle the exported symbols
+# - although elektra has support for symbol mangling (ELEKTRA_VARIANT) we can't
+#   make us of this because the exported symbols will also be generated by the
+#   build environment for the elektra-full build (magic is happening in exported_symbols)
+# - and it's impossible to remove the plugin from elektra-full
```

Comment:
why is this impossible? Maybe its even useful to exclude them, e.g. kdb-full starting the java interpreter causes a lot of trouble.